http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AzurePubsubHandler.scala

```diff
@@ -101,8 +105,9 @@ class AzurePubsubHandlerInterp[F[_]: Parallel](
         case x: CloudContext.Azure => F.pure(x.value)
       }
       hcName = RelayHybridConnectionName(params.runtime.runtimeName.asString)
-      primaryKey <- azureRelay.createRelayHybridConnection(params.relayeNamespace, hcName, cloudContext)
+      primaryKey <- azureRelay.createRelayHybridConnection(params.landingZoneResources.relayNamespace, hcName, cloudContext)
       createDiskAction = createDisk(params, auth)
+      // TODO PR question: can we use one of the LZ subnets for this? Possibly the compute subnet?
```

Comment:
Yes! We shouldn't explicitly create a network anymore, we should use the compute subnet I think.